lib/venue/venuedetails.dart

```diff
@@ -9,6 +10,10 @@ class VenueDetailsView extends StatelessWidget {
   {
     final theme = Theme.of(context);
 
+    Firestore.instance.collection('conference_info').getDocuments().asStream().listen(
```

Comment:
have you considered wrapping this with a service so that you can swap Firestore with whatever in the future?